msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -372,29 +380,108 @@ GET /v1/meetings/{meetingId}
 
 The JSON response body for Meeting Details API is as follows:
 
-```json
-{ 
-   "details": { 
-        "id": "meeting ID", 
-        "msGraphResourceId": "", 
-        "scheduledStartTime": "2020-08-21T02:30:00+00:00", 
-        "scheduledEndTime": "2020-08-21T03:00:00+00:00", 
-        "joinUrl": "https://teams.microsoft.com/l/xx", 
-        "title": "All Hands", 
-        "type": "Scheduled" 
-    }, 
-    "conversation": { 
-            "isGroup": true, 
-            "conversationType": "groupchat", 
-            "id": "meeting chat ID" 
-    }, 
-    "organizer": { 
-        "id": "<organizer user ID>", 
-        "aadObjectId": "<AAD ID>", 
-        "tenantId": "<Tenant ID>" 
+* **Scheduled meetings:**
+
+    ```json
+
+    {
+       "details":  { 
+                        "id": "<meeting ID>", 
+                        "msGraphResourceId": "MSowYmQ0M2I4OS1lN2QxLTQxNzAtOGZhYi00OWJjYjkwOTk1YWYqMCoqMTk6bWVldGluZ19OVEkyT0RjM01qUXROV1UyW", 
+                        "scheduledStartTime": "2022-04-24T22:00:00Z", 
+                        "scheduledEndTime": "2022-04-24T23:00:00Z", 
+                        "joinUrl": "https://teams.microsoft.com/l/xx", 
+                        "title": "All Hands", 
+                        "type": "Scheduled" 
+                    }, 
+        "conversation": { 
+                            "isGroup": true, 
+                            "conversationType": "groupchat", 
+                            "id": "meeting chat ID" 
+                        }, 
+        "organizer": { 
+                        "id": "<organizer user ID>", 
+                        "aadObjectId": "<AAD object ID>",
+                        "objectId": "<organizer object ID>",
+                        "tenantId": "<Tenant ID>" 
+                    }
+    } 
+    ```
+
+* **One-on-one calls:**
+
+    ```json
+    {
+        "details": {
+                        "id": "<meeting ID>",
+                        "type": "OneToOneCall"
```

Comment:
The documentation below for details.type contains: Adhoc, Broadcast, MeetNow, Recurring, Scheduled, Unknown

For the other values (Adhoc, Broadcast, Recurring, Unknown), when are those used? Are they still valid?